ui/component/notification/view.jsx

```diff
@@ -140,35 +175,61 @@ export default function Notification(props: Props) {
           'notification__wrapper--unread': !is_read,
         })}
       >
-        <div className="notification__icon">{icon}</div>
+        <NavLink {...navLinkProps}>
+          <div className="notification__icon">{icon}</div>
+        </NavLink>
 
         <div className="notification__content-wrapper">
           <div className="notification__content">
             <div className="notification__text-wrapper">
-              {!isCommentNotification && (
-                <div className="notification__title">
-                  <LbcMessage>{notification_parameters.device.title}</LbcMessage>
-                </div>
-              )}
+              <NavLink {...navLinkProps}>
+                {!isCommentNotification && (
+                  <div className="notification__title">{title}</div>
+                )}
 
-              {isCommentNotification && commentText ? (
-                <>
-                  <div className="notification__title">
-                    <LbcMessage>{notification_parameters.device.title}</LbcMessage>
-                  </div>
-                  <div title={commentText} className="notification__text mobile-hidden">
-                    {commentText}
-                  </div>
-                </>
-              ) : (
-                <>
-                  <div
-                    title={notification_parameters.device.text.replace(/\sLBC/g, ' Credits')}
-                    className="notification__text"
-                  >
-                    <LbcMessage>{notification_parameters.device.text}</LbcMessage>
-                  </div>
-                </>
+                {isCommentNotification && commentText ? (
+                  <>
+                    <div className="notification__title">{title}</div>
+                    <div title={commentText} className="notification__text mobile-hidden">
+                      {commentText}
+                    </div>
+                  </>
+                ) : (
+                  <>
+                    <div
+                      title={notification_parameters.device.text.replace(/\sLBC/g, ' Credits')}
+                      className="notification__text"
+                    >
+                      <LbcMessage>{notification_parameters.device.text}</LbcMessage>
+                    </div>
+                  </>
+                )}
+              </NavLink>
+
+              {isCommentNotification && (
+                <div className="notification__reactions">
+                  <Button
+                    label={__('Reply')}
+                    className="comment__action"
+                    onClick={handleCommentReply}
+                    icon={ICONS.REPLY}
+                  />
+                  <CommentReactions uri={notificationTarget} commentId={notification_parameters.dynamic.hash} />
```

Comment:
`CommentReactions` will show 0 reactions each time unless you've visited the comment before.
One suggestion is to collect a list of comment IDs at one level up and fetch all reactions in one shot.